.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: php
+
+sudo: false
+
+cache:
+    directories:
+        - $HOME/.composer/cache/files
+
+php:
+    - 5.4
+    - 5.5
+    - 5.6
+    - 7.0
+    - hhvm
+
+matrix:
+    include:
+        - php: 5.4
+          env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
```

Comment:
``SYMFONY_DEPRECATIONS_HELPER`` is a preparation for the inclusion of the Symfony PHPUnit bridge to catch deprecations in the testsuite (it was there in the config I copied, so I haven't removed it to add it later)